public/app/features/alerting/unified/api/incidentsApi.ts

```diff
@@ -7,8 +7,7 @@ interface IncidentsPluginConfigDto {
   isIncidentCreated: boolean;
 }
 
-const getProxyApiUrl = (path: string) =>
-  `/api/plugins/grafana-incident-app/resources/${SupportedPlugin.Incident}${path}`;
+const getProxyApiUrl = (path: string) => `/api/plugins/${SupportedPlugin.Incident}/resources${path}`;
```

Comment:
Does nothing flow through MSW that would have caught this? 🤔 e.g. an unhandled request in the tests?

Reply:
I don't think so....  basically we need to check that this endpoint is correct at the BE side